### ISSUE TITLE
Styles are not loading in Oqtane Server

### ISSUE THREAD

**Issue by Lingala-Joshna** [oqtane/oqtane.framework#2035]
When added styles resources in the project, run the project using Oqtane server, the styles are not loading in the Oqtane browser module

We have added reference for Syncfusion Styles in the HostResources.cs file in Server project

```
public class HostResources : IHostResources
    {
        public List<Resource> Resources => new List<Resource>()
         {
             // Only register .css files.
             // The JavaScript files will automatically be pulled
             // from the _content/syncfusion.blazor directory
             // so they do not need to be registered here.
             new Resource {
             ResourceType = ResourceType.Stylesheet,
             Url = "_content/Syncfusion.Blazor/" +
             "styles/bootstrap5.css" },
             new Resource {
             ResourceType = ResourceType.Stylesheet,
             Url = "_content/Syncfusion.Blazor/" +
             "styles/tailwind-dark.css" },
             new Resource {
             ResourceType = ResourceType.Script,
             Url = "https://kit.fontawesome.com/a076d05399.js" }
         };
    }
```
Copied the Scripts and styles files in server project and also copied to the Oqtane.server using debug.cmd

![image](https://user-images.githubusercontent.com/93309503/156161698-bbf204a6-3194-4a67-9b46-3da502d5e657.png)

![image](https://user-images.githubusercontent.com/93309503/156161602-e126e22b-c000-444e-98d5-b8f879d3edaa.png)

Output in oqtane server browser:

![image](https://user-images.githubusercontent.com/93309503/156163909-81495b98-6706-4989-a7ae-aec73281dc46.png)

**Replication Steps:**

- Download the zip file
- Extract the files and open the project in VisualStudio2022
- Run the sample using Oqtane server
- Styles are not reflecting in the output oqtane server browser

[Sample.SyncProj.zip](https://github.com/oqtane/oqtane.framework/files/8159388/Sample.SyncProj.zip)

**Comment by ghhv** [comment]
I was just playing with the default theme options, did you also install the new theme via the admin page (/admin/themes) and then select the theme from Site Settings (/admin/site)? I assume what you've done is only the download step.. Just guessing - it's my first day testing this..

**Comment by sbwalker** [comment]
this issue is resolved in #2045. You can also resolve it in the current version by registering your resources in your module using the Declaration = ResourceDeclaration.Global attribute:

    public class HostResources : IHostResources
    {
        public List<Resource> Resources => new List<Resource>()
        {
            new Resource {
                ResourceType = ResourceType.Stylesheet,
                Declaration = ResourceDeclaration.Global,
                Url = "_content/Radzen.Blazor/css/default.css" }
        };
    }

That being said, I do not believe that the IHostResources approach is the recommended way to register static resources - it was more of a temporary solution while a more robust solution was being developed for Oqtane. The problem with IHostResources is that it registers the assets globally ie. it includes the stylesheet and JavaScript references on every page for every site within your multi-tenant installation - which is generally not a good idea. Declaring the resources locally in a module is much better as it means they are lazy loaded as needed. An example of a local declaration is in your Index.razor file:

@code {
    public override SecurityAccessLevel SecurityAccessLevel => SecurityAccessLevel.Anonymous;

    public override List<Resource> Resources => new List<Resource>()
    {
        new Resource { ResourceType = ResourceType.Stylesheet, Url = "_content/Radzen.Blazor/css/default.css" }
    };

**Comment by beolafsen** [comment]
Thanks - this works fine
